### PR TITLE
Enable scripting of Eazy

### PR DESCRIPTION
**Pull request by profxj**
Mainly for building galaxy objects.

Still need to add a few docs.

### CODE REVIEW

**Comment by profxj** [comment]
And a test or 3

File: frb/galaxies/eazy.py [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+    # Uncertainty
+    cum_pzi = np.cumsum(pzi) / np.sum(pzi)
+    l68 = zgrid[np.argmin(np.abs(cum_pzi - 0.16))]
+    u68 = zgrid[np.argmin(np.abs(cum_pzi - (1 - 0.16)))]
+    #
+    return zphot, (u68-l68)/2.
+
+
+####################################################
+#  The following are taken from threedhst by Brummer
```
Comment (SunilSimha) [inline frb/galaxies/eazy.py:192]: Do consider adding the SED fit visualization functions from there as well.

File: frb/galaxies/eazy.py [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+        logfile (str):
+    """
+    _, param_file = eazy_filenames(input_dir, name)
+
+    # Find the eazy executable
+    path_to_eazy = spawn.find_executable('eazy')
+    if path_to_eazy is None:
+        raise ValueError("You must have eazy in your Unix path..")
+    # Run it!
+    command_line = [path_to_eazy, '-p', os.path.basename(param_file)]
```
[the remark this answers is not in the document]
Comment (profxj) [reply frb/galaxies/eazy.py]: done.

File: frb/data/analysis/EAZY/zphot.param.default [hunk 3]
```diff
@@ -0,0 +1,75 @@
+#### EAZY Default parameters
+
+## Filters
+FILTERS_RES          FILTER.RES.latest  # Filter transmission data
+FILTER_FORMAT        1                  # Format of FILTERS_RES file -- 0: energy-  1: photon-counting detector
+SMOOTH_FILTERS       n                  # Smooth filter curves with Gaussian
+SMOOTH_SIGMA         100.               # Gaussian sigma (in Angstroms) to smooth filters
+
+## Templates
+TEMPLATES_FILE       templates/br07_default.spectra.param # Template definition file
```
[the remark this answers is not in the document]
Comment (profxj) [reply frb/data/analysis/EAZY/zphot.param.default:10]: got it.

File: frb/data/analysis/EAZY/zphot.param.default [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+DUMP_TEMPLATE_CACHE  n                  # Write binary template cache
+USE_TEMPLATE_CACHE   n                  # Load in template cache
+CACHE_FILE           photz.tempfilt     # Template cache file (in OUTPUT_DIRECTORY)
+
+## Input Files
+CATALOG_FILE         REPLACE.cat # Catalog data file
+MAGNITUDES           n                  # Catalog photometry in magnitudes rather than f_nu fluxes
+NOT_OBS_THRESHOLD    -90                # Ignore flux point if <NOT_OBS_THRESH
+N_MIN_COLORS         5                  # Require N_MIN_COLORS to fit
```
[the remark this answers is not in the document]
Comment (profxj) [reply frb/data/analysis/EAZY/zphot.param.default:34]: now allowed

File: frb/galaxies/eazy.py [hunk 5]
```diff
@@ -0,0 +1,363 @@
...
+    with open(logfile, 'w') as f:
+        retval = subprocess.call(command_line, stdout=f, stderr=f, cwd=input_dir)
+
+
+def eazy_stats(zgrid, pzi):
```
[the remark this answers is not in the document]
Comment (profxj) [reply frb/galaxies/eazy.py:316]: I like calculating them myself so that I know what I did.  :)

File: frb/galaxies/eazy.py (hunk 1 above)
Comment (profxj) [reply frb/galaxies/eazy.py:338]: considered, but will pass for now.

**Comment by profxj** [comment]
@SunilSimha -- Ready for another pass.  Thanks for the feedback.

@almannin -- Please take a look as well.

**Comment by profxj** [comment]
And please confirm the tests pass for you, i.e.  run:

`py.test  --remote_data`

**Comment by profxj** [comment]
merging

**Pull request merged**